cli/main.go

```
@@ -82,6 +94,72 @@ func main() {
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("failed to download yamls: %s", resp.Status)
+	}
+
+	home, _ := os.UserHomeDir()
+	dest := filepath.Join(home, ".k3sd", "yamls")
+	os.MkdirAll(dest, 0700)
```

golangci-lint (1.64.8)

Error return value of `os.MkdirAll` is not checked

(errcheck)